cf_xarray/criteria.py

```diff
@@ -8,6 +8,34 @@
 import re
 from typing import MutableMapping, Tuple
 
+# KMT: Not finished, but as draft
```

Comment:
This is not needed. We can create one later for standard_name aliases Otherwise, I intend to leave this up to a user to create.

Reply:
Ok I removed that.